Nuke Ops Ordnance Lab usability fixes

Pandarsenic opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Examination of the new Nuke Ops Ordnance Lab on Campbell yielded a number of visible usability problems. 

Nuclear Ops Ordnance lab now has...
an air alarm + a console for the burn chamber and corresponding sensor (EDIT: The air alarm now starts unlocked so they don't have to hack into it).
![image](https://user-images.githubusercontent.com/101627558/164157179-5042e803-ffbf-4ae5-83e1-f6d37dbb75b7.png)
a burn chamber without starting gasses
a scrubber to empty tanks
one RPD to rearrange pipes as needed (EDIT: And a Gas Analyzer so they don't have to run all the way over to the YouTool for one).
four Tier-2 Lasers to make the freezers properly usable for bombmaking
![image](https://user-images.githubusercontent.com/101627558/164157281-cb5498ac-df1f-43d5-b0d9-890b6c169144.png)
two Tier-2 Canisters to receive hot gasses from the burn chamber without immediately rupturing
a waste pipe for the freezers so they can vent their heat
![image](https://user-images.githubusercontent.com/101627558/164156944-2d730009-56cc-4b55-915e-9f5f908d1d73.png)
two Tier-2 Canisters to receive hot gasses from the burn chamber without immediately rupturing
a waste pipe for the freezers so they can vent their heat (EDIT: Adjusted snow turf with the passive vent so that it actually provides gas to the waste piping)
two more O2 canisters so they can make a usable amount of tritium and have O2 left to cool for payload gas.

Removed N2 and N2O, which are only used in exotic gas production
Replaced two walls with floors to make room for the air alarm and burn chamber console.
Shortened a couple of pipes to put things in the spaces freed up by doing so.

Wall camera, light positions, decals adjusted accordingly. (EDIT: Lights tampered with further)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

The Nukie Ordnance Lab should be harder to use than the Station Ordnance Lab, but it does need the bare minimum functional equipment for operatives to actually make Tritium with it.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
fix: Nuclear Operative Ordnance Lab now has minimum functional equipment.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Pandarsenic committed:
Nuke Ops Ordnance Lab usability fixes

Nuclear Ops Ordnance lab now has an air alarm; a console for the burn chamber and corresponding sensor; a burn chamber without starting gasses; a scrubber to empty tanks; one RPD to rearrange pipes as needed; four Tier-2 Lasers to make the freezers properly usable for bombmaking; two Tier-2 Canisters to receive hot gasses from the burn chamber without immediately rupturing; a waste pipe for the freezers to so they can vent their heat; and 2 more O2 canisters so they can make a usable amount of tritium. Removed N2 and N2O, which are only used in exotic gas production. Replaced two walls with floors to make room for the air alarm and burn chamber console.

san7890 reviewed (changes requested):
These changes were requested by me since Pandarseric mentioned there being more than a few oversights with the ordnance portion of the nuke ops base. I attempted to retain as much as the original design intent (to my understanding) as possible, but some things just did not work without the proper mechanic-based equipment.

With that all said and done, here's the review:

<details>
<summary>Click Me For Images</summary>

I already discussed most of this in-depth in a thread in #mapping-general on the /tg/station13 Discord.

![image](https://user-images.githubusercontent.com/34697715/164157891-b4f7d999-b53b-4a98-87c3-cfed55f6d1f0.png)

Mapping goofs, just delete the erratic air alarm and console.

![image](https://user-images.githubusercontent.com/34697715/164158349-0dfb3f3f-abca-4945-a7cd-1af60896dd3c.png)

Shows up like this:

![image](https://user-images.githubusercontent.com/34697715/164159804-1bd0aa23-9c56-4802-9e7a-e1868fe82723.png)

I think we were misleading you in the thread- please make these two tiles `/turf/open/misc/asteroid/snow/airless` (or was it icemoon's subtype? I'll double-check, but should definitely be on turf/open.

![image](https://user-images.githubusercontent.com/34697715/164158429-802ba3ac-911e-4218-971d-e88d32317331.png)

**Here as well, please make these two tiles `/turf/open/misc/asteroid/snow/airless`. Potato/LemonInTheDark will eat me for fucking breakfast if you don't.**

![image](https://user-images.githubusercontent.com/34697715/164159355-82d04953-0934-40ca-b5dd-29694ce7b0fd.png)

Lighting is a bit quirky- could you replace these with the "tube" lights? I know that the areas are set up to have different lighting in each room, so experiment a bit if you wouldn't mind.

![image](https://user-images.githubusercontent.com/34697715/164159509-31f2bf0a-c129-4b9f-a866-98037ffc068a.png)

For the injector we have in the nukie's mixing base, could you please edit the `machine id_tag` to the "nukiebase" we've been using? That should work- and we shouldn't have this issue after that.

![image](https://user-images.githubusercontent.com/34697715/164159675-0b5f0abb-995f-43fc-bcd3-67aa203793fd.png)

I'm not sure how much I trust this... it says that's a blind spot to the camera? Huh? Feel free to add more cameras, but I think it's just not correct.

![image](https://user-images.githubusercontent.com/34697715/164159884-28329ea1-b08b-42e0-9b58-95509c78b283.png)

This air alarm should be unlocked, like the standard ones at Ordnance. Let me know if you need help finding the subtype.
</details>

Please feel free to correct me if I am wrong about any of the atmospherics nuances. Other than that, great work with this change. I personally feel it retains some of the original feelings, while still adding the needed changes. It's always a good skill to have to be able to identify potential/known issues with something, and be willing to help resolve them. Also, if you don't fix the thing with the icemoon turfs on the passive vents I think LemonInTheDark will blow a fucking vein.

I marked your PR as a "Draft" so that we can sorta remember the status of each PR we currently have on the repo. Feel free to mark it "Ready for Review" when the points in this review have been addressed.

Pandarsenic commented:
Testing a couple of those things local real quick before moving on.

Pandarsenic committed:
Requested Updates from Maptainer to Syndicate Ordnance Lab

Stray machines deleted in Xenobio... and the bathroom...?
Turfs under passive vents adjusted. It looks like it gets about 25% cooling efficiency, but I'm sure the Flukies will manage.
Lights tweaked to tubes. Much less moody, but much better visibility.
Injector fixed on consultation in Discord; improper subtype replaced with proper subtype, easy money.
Camera is blind because of the tinted windows; I don't want to change those without the original mapper's go-ahead, but we can see what we can do if it's a meaningful problem.
Air alarm swapped to the unlocked subtype.

Pandarsenic commented:
Stray machines deleted in Xenobio... and the bathroom...?
Turfs under passive vents adjusted. It looks like it gets about 25% cooling efficiency, but I'm sure the Flukies will manage.
Lights tweaked to tubes. Much less moody, but much better visibility.
Injector fixed on consultation in Discord; improper subtype replaced with proper subtype, easy money.
Camera is blind because of the tinted windows; I don't want to change those without the original mapper's go-ahead, but we can see what we can do if it's a meaningful problem.
Air alarm swapped to the unlocked subtype.

Zytolg commented:
For balance reasonings, I omitted the RPD from the original PR. If the ability to work on/with pipes is absolutely a necessity, then I'd include a few pipes with whichever is required. I consulted a toxins expert on the pipe placements pre-included and they told me it was serviceable. 
No pipe tools were included for 2 reasons:
1) These will be used to build out into the laboratory hallway and I wanted to avoid that for thematics.
2) Those who are atmos inclined will absolutely find a way to jump beyond what is intended for the lab. It's already possible with the current setup, but at least that's a controlled environment. 

I have no qualms with making the lab itself serviceable to the minimal specifications of creating maxcaps. I am hugely cautious about burring the lines between what is intended as a focused Toxins Lab, and what could become a Gas Manufacturing facility. In my own experience, I very much believe the RPD to be that distinction. 

<details><summary> Thoughts after consulting Ghilker </summary>
A dozen minutes or so later, and I talked it out with Ghilker and have a decent understanding of what is possible with the current method, vs what can be possible with an RPD. Now, the main catalyst for this becoming a Gas Manufacturing facility is the presence of N2O, which facilitates the creation of BZ: an atmos Catalyst of sorts. This condition has already been met, however is limited drastically by the factor of time due to the current setup. Now, with the whole point of the labs being to function as a reward mechanism towards player experience, I've decided that I want to leave the N2O in.

With the BZ factor out of the way, this meant that I really had to confirm what the presence of an RPD facilitates. Saving time is naturally a big one here, but what else? Well, an RPD facilitates the construction of any pipe in the game provided you can satisfy the material upkeep. Since the Nukie Base (and Ship) feature an abundance of Material, running out shouldn't be a condition that is easily met. Now, the Nukie Base is situated on Freyja, aka Icemoon. Icemoon turfs spawn a cold mixture of Oxygen and Nitrogen, but aren't limited to simply those two. In small amounts, one may chance to discover that Miasma, CO2, and Plasma are also in the mix. It's that last gas, Plasma, where I want to focus on. An abundant RPD facilitates the swift harvest of the chance Plasma spawn. I see players going this far to harvest Plasma. 

Now, if someone is stubborn enough to take a scrubber, unwrench a few pipes, and go do the same thing to fill up a canister, I'm not going to stop them. I've already said that I want to reward game knowledge. The distinction here, is that I want the laboratories to encourage their primary usage over possible secondary usages. At the end of this research, my conclusion remains that the inclusion of an RPD muddies this distinction.
</details>

Mapping features such as this is such a nuanced process. Given something especially as powerful as Nukeops, you want to be careful to toe-the-line exactly. I put in a brute 20 (down from 40) katana because I knew that it is inferior to the options present in the uplink. It's only there for flavor though I absolutely know someone will use it for the style points. Trying out dumb things is an option, but there's a tug away from it so that the distinction is clear. Ensuring that things work is great and you've done good work here. Don't think that I'm out to get you because you haven't. Instead, consider the RPD part, and be the one to decide weather or not keeping it in is the correct decision. I had already made mine with the original PR, and now it's your turn to consider the facts.

I leave it in your hands, mapper.

Pandarsenic committed:
Syndie Ordnance: Moves pipes, adjusts freezers for easier access

Pandarsenic commented:
I'm leaving the RPD for now, but some tests in trying to produce a maxcap without using the piping locally had some odd results. The chamber doesn't burn as hot as chambers on the station that seem identical in every respect but their walls and the positioning of their blast doors.

Until/unless I can get a maxcap without the RPD, I think - with the N2O gone and the time-intensiveness of most gas farming - the RPD's presence isn't a serious threat in most circumstances, at least for now, but I'm open to anyone who can demonstrate that it's able to produce a threat more significant than what Nuke Ops should typically have in a timely manner.

Pandarsenic committed:
Seeing if turning the freezers is what upset Map Diff Bot

:(

Pandarsenic committed:
Syndie Ordnance Final(?) tuning

Turns freezers for easier canister access and to keep the waste pipes from being visible under the window. Re-adds corner wall to unsquare the room and bring the air alarm closer to the monitor. Adjusts lights and decals accordingly.

Pandarsenic commented:
God DAMN it the pipes are visible under the Bio-Weapon Lab windows. Whatever, it'll be fine unless someone wants me to change it for aesthetics.
